manim/mobject/types/vectorized_mobject.py

```diff
@@ -784,7 +791,7 @@ def insert_n_curves(self, n):
             self.append_points([new_path_point])
         return self
 
-    def insert_n_curves_to_point_list(self, n, points):
+    def insert_n_curves_to_point_list(self, n: int, points: List[Vector]):
```

Comment:
Requires `len` so Sequence; can return either self or array so Union
```suggestion
    def insert_n_curves_to_point_list(self, n: int, points: typing.Sequence[Vector]) -> typing.Union["SVGMobject", np.ndarray]:
```

Reply:
`insert_n_curves_to_point_list` never returns an SVGMobject

Reply:
my bad